iina/PlaylistViewController.swift

```diff
@@ -444,14 +444,23 @@ class PlaylistViewController: NSViewController, NSTableViewDataSource, NSTableVi
   var selectedRows: IndexSet?
 
   func menuNeedsUpdate(_ menu: NSMenu) {
-    var indexSet = playlistTableView.selectedRowIndexes
-    if playlistTableView.clickedRow >= 0 {
-      indexSet.insert(playlistTableView.clickedRow)
-    }
-
-    selectedRows = indexSet
+    let selectedIndexs: IndexSet = {
```

Comment:
This code looks very similar to that in HistoryWindowController; is there any way to factor it out?